core/src/main/java/com/scalar/db/transaction/consensuscommit/TransactionResult.java

```diff
@@ -106,12 +108,17 @@ public Map<String, Column<?>> getColumns() {
     return result.getColumns();
   }
 
+  @Nullable
   public String getId() {
     return getText(Attribute.ID);
   }
 
   public TransactionState getState() {
-    return TransactionState.getInstance(getInt(Attribute.STATE));
+    int state = getInt(Attribute.STATE);
```

Comment:
Although `getInt` returns 0 for a null value, 
but should it be slightly more appropriate/readable to use `isNull` since it actually checks if the value is null?

Reply:
That's right. Thanks. Fixed in d5ff7af.